prime-router/docs/onboarding-users/migrating-receivers.md

```diff
@@ -0,0 +1,248 @@
+# How to Migrate an existing receiver to the UP
+
+## Welcome
+
+The goal of this documentation is to provide a guide on how to migrate existing covid receivers to the Universal pipeline.
+
+The main goal of receiver onboarding is to establish a connection to send data from ReportStream to the STLT.  
+In order to do so there are multiple receiver configurations that need to be configured.
+
+## Pre-Migration
+* Identify if the STLTs transport is set-up in staging [How to check if STLTs transport is working](#how-to-check-if-stlts-transport-is-working)
+    *  Determine if receiver's SFTP server needs to be whitelisted by the CDC [How to check if STLT needs to be whitelisted](#how-to-check-if-stlt-needs-to-be-whitelisted)
+* During kick-off call -Determine if we need to send to the existing endpoint and if new credentials are needed for existing endpoint
+
+## Table of Contents
+1. [Fetch current receiver settings](#1-fetch-current-receiver-settings)
+2. [Update receiver settings to route messages through UP](#2-update-receiver-settings-to-route-messages-through-up)
+3. [Send test message from SimpleReport to STLT](#3-send-test-message-from-simplereport-to-stlt)
+4. [Set-up Receiver Transforms](#4-set-up-receiver-transforms)
+5. [Compare Covid and UP messages](#5-compare-covid-and-up-messages)
+6. [Testing in Staging](#6-testing-in-staging)
+7. [Move setting to production](#7-move-settings-to-production)
+
+### 1. Fetch current receiver settings
+To start migrating the receiver to the UP their most current settings need to be retrieved from production. 
+After retrieving the settings they will be updated to be able to route a message through the UP.
+
+* To begin create a new branch in git for your changes.
+* Fetch the STLTs production settings by running this command: `./prime multiple-settings get -f tx-doh --output tx-doh.yml`
```

Comment:
maybe genericize this to:

 "run this command: './prime multiple-settings get -f <filter-criteria> --output <file-location-for-output>

example - to get the most current receiver settings for organization tx-doh:
'./prime multiple-settings get -f tx-doh --output tx-doh.yml'